### PR TITLE
Make keyboard-only focusable for: `<Checkbox> <RadioButton> <RichTextAreaButton> <ToggleSwitch>`

### PR DESCRIPTION
**Pull request by hepiyellow**

### CODE REVIEW

File: src/Backoffice/Button/Button.e2e.js [hunk 1]
```diff
@@ -4,12 +4,14 @@ import autoExampleDriver from 'wix-storybook-utils/AutoExampleDriver';
...
 
+const NO_DESCRIPTION = '';
+
 describe('Backoffice Button', () => {
   const storyUrl = getStoryUrl('5. Buttons', '5.1 Standard');
   const driver = buttonTestkitFactory({dataHook: 'storybook-button'});
 
   // Specific as opposed to 'Generic' tests like the Focusable tests.
-  describe('Specific', () => {
+  describe(NO_DESCRIPTION, () => {
```
Comment (argshook) [inline src/Backoffice/Button/Button.e2e.js]: huh? why?

File: src/Checkbox/Checkbox.protractor.driver.js [hunk 2]
```diff
@@ -1,15 +1,43 @@
...
+    clickableElements: [checkboxElement, childrenElement]
+  });
+
+  const publicDriver = {
+    element: () => element,
+    click: () => element.click(),
+    getLabel: () => element.$(`label`),
+    getInput: () => element.$(`input`),
+    isChecked: () => element.$(`input`).isSelected(),
+    isDisabled: async () => await element.$(`input`).getAttribute('disabled') !== null,
```
Comment (argshook) [inline src/Checkbox/Checkbox.protractor.driver.js]: i suppose if we stop relying on protractor control flow and do `async`/`await` then need to do it everywhere? like, in `click`, `getLabel`... methods also

File: src/Checkbox/Checkbox.scss [hunk 3]
```diff
@@ -2,13 +2,27 @@
...
+  outline: none;
+  
   label {
     display: flex;
     cursor: pointer;
     justify-content: center;
     align-items: center;
   }
+
+  &#{$focusable-focus-visible} {
```
Comment (argshook) [inline src/Checkbox/Checkbox.scss]: i reeeally vote for explicit import for this one

File: src/Checkbox/Checkbox.protractor.driver.js [hunk 4]
```diff
@@ -1,15 +1,43 @@
-import {isFocused} from '../test-common';
 import styles from './Checkbox.scss';
+import focusableDriverFactory from '../common/Focusable/Focusable.protractor.driver';
+import {mergeDrivers} from '../test-common';
```
Comment (argshook) [inline src/Checkbox/Checkbox.protractor.driver.js]: i don't know... i get the idea but it feels as unnecesarry complication. if we want to make driver which is not exposed... then let's just not expose it. `merge`, `flatten`, symbol.. wont it make harder to maintain without much benefit? maybe i miss something, but for what i understand, wouldn't do any of that

File: src/RadioGroup/RadioGroup.e2e.js [hunk 5]
```diff
@@ -37,29 +39,71 @@ describe('RadioGroup', () => {
...
-        expect(radioGroupDriver.isRadioFocused(0)).toBe(false);
-        await pressTab();
-        expect(radioGroupDriver.isRadioFocused(0)).toBe(true);
+        // TODO: replace with forEachAsync
+        for (let index = 0; index < NUM_OF_BUTTONS_IN_EXAMPLE; index++) {
+          const driver = flattenInternalDriver(groupDriver.getButtonDriver(index));
+          await expectNotFocused(`button ${index} - before`, driver);
+          await pressTab();
+          await expectFocusedByKeyboard(`button ${index} - after`, driver);
+          index === 0 && await eyes.checkWindow(`button ${index} with focus-visible`);
```
Comment (argshook) [inline src/RadioGroup/RadioGroup.e2e.js]: move out from loop and no need for `index === 0`

File: src/Backoffice/Button/Button.e2e.js (hunk 1 above)
Comment (hepiyellow) [reply src/Backoffice/Button/Button.e2e.js]: I need to separate the Generic tests so they run in a separate `describe` block, so I have control over the `beforeEach`.
So the non-generic tests should be wrapped with a `describe` block.
I used to call the `Specific` and `Generic`, but it looks bad in the combined test names "Button Specific ....", so I decided to have the "Specific" tests with no description, and only have "Generic" for the generic test suites.

File: src/Checkbox/Checkbox.protractor.driver.js (hunk 2 above)
Comment (hepiyellow) [reply src/Checkbox/Checkbox.protractor.driver.js]: The driver doesn't need to do async/await for 1 liner methods. this `isDisabled` is a mistake. not mine. I didn't fix it. I can fix it.

File: src/Checkbox/Checkbox.scss (hunk 3 above)
Comment (hepiyellow) [reply src/Checkbox/Checkbox.scss]: I can do that.

File: src/Checkbox/Checkbox.protractor.driver.js (hunk 4 above)
Comment (hepiyellow) [reply src/Checkbox/Checkbox.protractor.driver.js]: I see your pain. 
We'll discuss it in the next weekly.

**Pull request merged**